python/dgl/distgnn/partition/libra_partition.py

```diff
@@ -59,8 +59,8 @@ def libra_partition(num_community, G, resultdir):
     3. The folder also contains a json file which contains partitions' information.
     """
 
-    num_nodes = G.number_of_nodes()  # number of nodes
-    num_edges = G.number_of_edges()  # number of edges
+    num_nodes = G.num_nodes()  # number of nodes
```

Comment:
The number_of_nodes is already an alias of num_nodes. https://github.com/dmlc/dgl/blob/0dd4f76745f5e5e475eb664c18262abbe047d635/python/dgl/distributed/dist_graph.py#L964

@BarclayII can you double check this change is OK to go?